src/deepsparse/loggers/constants.py

```diff
@@ -43,6 +41,15 @@ class MetricCategories(Enum):
     DATA = "data"
 
 
+@dataclass(frozen=True)
+class SystemGroups:
+    # Pipeline System Groups
+    PREDICTION_LATENCY: str = "prediction_latency"
+    # Server System Groups
+    REQUEST_DETAILS: str = "request_details"
+    RESOURCE_UTILIZATION: str = "resource_utilization"
```

Comment:
Thought, maybe instead of adding a comment that also needs to be maintained, would renaming to `PIPELINE_PREDICTION_LATENCY`, AND `SERVER_REQUEST_DETAILS` be more meaningful?

Reply:
I do not like to make those variables long tbh. We will be composing quite complex strings using those names, my focus is to keep them short.

Reply:
caution against too short, but this looks like a good balance

Reply:
Approving for now, but my main concern is readability and context over short variable names, if someone wants to check which system group belongs to Pipeline they would have to come to this class and read a comment which could quickly go out of date.

Reply:
Also if these are constants, wouldn't an Enum be better?

Reply:
I hate using Enums for the awkward process of accessing them ;)